src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -126,9 +152,27 @@ protected override void PostPersist(ApplicationEngine engine)
 
             int index = (int)(engine.Snapshot.PersistingBlock.Index % (uint)ProtocolSettings.Default.CommitteeMembersCount);
```

Comment:
I think taht `ShouldRefreshCommitee` should be equal to this count.
```csharp
private bool ShouldRefreshCommittee(uint height) => height % (ProtocolSettings.Default.CommitteeMembersCount + ProtocolSettings.Default.ValidatorsCount) == 0;
```

Reply:
This is because
 _" The voters who voted for a consensus node are assigned 2/28 of this part according to the voting weight, and the voters who voted for a committee member (non-consensus node) are assigned 1/28 of this part according to the voting weight."_

Reply:
But if we refresh the list every 28 blocks, and we reward them with %21 there are 7 blocks where the first nodes will be rewarded if they lost the committee, the rest of the commitee haven't got this benefict.

Reply:
> there are 7 blocks where the first nodes will be rewarded if they lost the committee

The 7 nodes will not lost until the end of this epoch.